### ISSUE TITLE
DText: don't convert <small>and <sub> to [tn]

### ISSUE THREAD

**Issue by NamelessContributor** [danbooru/danbooru#5569]
https://github.com/danbooru/danbooru/blob/c5546b862fcf0fcf216a88bc3547a3b849fdf47b/app/logical/d_text.rb#L381-L382
Translation notes should only be used to differentiate our additions from the original commentary, not for formatting the original commentary.
Additionally this can mess up the original formatting, see for example [post #6606674](https://danbooru.donmai.us/posts/6606674).

**Comment by evazion** [comment]
>Translation notes should only be used to differentiate our additions from the original commentary, not for formatting the original commentary.

The `[tn]` tag has no semantic meaning. It's just a formatting tag that means "small grey text".

A `[small]` tag could be added to distinguish small text from translation notes, but people would just mix them up or come to use them interchangeably because visually they would look the same.

This is similar to how HTML has `<b>`, `<i>`, `<em>`, and `<strong>` tags, and how `<b>` vs. `<strong>` and `<i>` vs. `<em>` are supposed to have different semantic meanings and be used for different purposes, but in reality they just mean "bold" and "italics" and any distinctions between them are meaningless.

**Comment by NamelessContributor** [comment]
Yes, but for the purpose of archiving the original commentary it is useful to be able to clearly distinguish between "translation notes" and "small text that's part of the commentary".
